### PR TITLE
add video link for legend

### PR DESCRIPTION
**Pull request by CovMat**

### CODE REVIEW

**Comment by seisman** [comment]
Maybe we can embed the video in the document? See https://ebf-contribute-guide.readthedocs.io/zh_CN/latest/rest-syntax/pic-video.html#id3
```
<iframe src="//player.bilibili.com/player.html?aid=447610668&bvid=BV13j411q79A&cid=1240906532&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```

**Comment by CovMat** [comment]
好像不支持。是我哪里写的不对吗？

File: source/tutorial/legend.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 添加图例
 ========
 
+:视频讲解: 
+
+.. raw:: html
+
+<iframe src="//player.bilibili.com/player.html?aid=447610668&bvid=BV13j411q79A&cid=1240906532&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```
Comment (seisman) [inline source/tutorial/legend.rst:8]:
```suggestion
.. raw:: html

   <iframe src="//player.bilibili.com/player.html?aid=447610668&bvid=BV13j411q79A&cid=1240906532&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```

File: source/tutorial/legend.rst [hunk 2]
```diff
@@ -1,6 +1,14 @@
 添加图例
 ========
 
+:视频讲解: 
+
+.. raw:: html
+
+   <iframe src="//player.bilibili.com/player.html?aid=447610668&bvid=BV13j411q79A&cid=1240906532&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```
Comment (seisman) [inline source/tutorial/legend.rst:8]:
```suggestion
   <iframe src="//player.bilibili.com/player.html?aid=447610668&bvid=BV13j411q79A&cid=1240906532&page=1" width="640" height="360" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```

File: source/tutorial/legend.rst [hunk 3]
```diff
@@ -1,6 +1,14 @@
 添加图例
 ========
 
+:视频讲解: 
+
+.. raw:: html
+
+   <iframe src="//player.bilibili.com/player.html?aid=447610668&bvid=BV13j411q79A&cid=1240906532&page=1" width="640" height="360" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```
Comment (seisman) [inline source/tutorial/legend.rst:8]:
```suggestion
   <iframe src="//player.bilibili.com/player.html?aid=447610668&bvid=BV13j411q79A&cid=1240906532&page=1" width="800" height="450" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```

File: source/tutorial/legend.rst [hunk 4]
```diff
@@ -1,6 +1,14 @@
 添加图例
 ========
 
+:视频讲解: 
+
```
Comment (seisman) [inline source/tutorial/legend.rst:5]:
```suggestion
```

**Comment by seisman** [comment]
https://gmt-china.github.io/sitepreview/gmt-china/GMT_docs/add_video_link_for_legend/tutorial/legend/

@CovMat Now it looks good to me. Are you OK with it?

**Review by seisman — APPROVED** [review]

**Comment by CovMat** [comment]
> https://gmt-china.github.io/sitepreview/gmt-china/GMT_docs/add_video_link_for_legend/tutorial/legend/
> 
> @CovMat Now it looks good to me. Are you OK with it?

可以啊

**Pull request merged**